### PR TITLE
Add Comma icon

### LINKED ISSUE #11033: Request: Comma
### Brand Name

Comma

### Website

https://www.comma.ai/

### Popularity Metric

[similarweb](https://www.similarweb.com/website/comma.ai/#overview) 
> 227,534

they also maintain [openpilot](https://github.com/commaai/openpilot)
> 48.2 ✨

### Official Resources for Icon and Color

I believe their main color is green, judging from all the green used in their events and website. I grabbed the following from the header of their website: 51FF00

As with the logo i used the vector found in their app repository:
https://github.com/commaai/connect/blob/master/public/icon-vector.svg

### Additional Comments

comma

### PR DESCRIPTION
**Pull request by 21st-centuryman**
**Issue:** closes #11033

**Popularity metric:**

[similarweb](https://www.similarweb.com/website/comma.ai/#overview)

> 227,534

they also maintain [openpilot](https://github.com/commaai/openpilot)

> 48.2 ✨

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I believe their main color is green, judging from all the green used in their [events](https://www.youtube.com/watch?v=u16msbVHNWk) and [website](https://www.comma.ai/). I grabbed the following from the header of their website: 51FF00

As with the logo i used the vector found in their app repository:
https://github.com/commaai/connect/blob/master/public/icon-vector.svg

image

### CODE REVIEW

**Review by uncenter — CHANGES_REQUESTED** [review]
> As with the logo i used the vector found in their app repository: https://github.com/commaai/connect/blob/master/public/icon-vector.svg

This should be listed as the source then?

**Comment by 21st-centuryman** [comment]
@uncenter I am not sure if it could apply here. I was confused by all other companies having the official website as their source, i could probably find a better source if thats what you want.

**Comment by uncenter** [comment]
> @uncenter I am not sure if it could apply here. I was confused by all other companies having the official website as their source, i could probably find a better source if thats what you want.

https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines

**Comment by 21st-centuryman** [comment]
@uncenter  thank you for your explanation, this has now been fixed.

**Review by uncenter — APPROVED** [review]
Seeing a slight difference when compared to the original with the difference tool near the bottom of the comma symbol though it looks fine in general.

**Comment by 21st-centuryman** [comment]
Sorry i didnt realize i accidentally force pushed. i have a new icon in the pr
@uncenter

**Comment by uncenter** [comment]
I'd like to review this again but something is up with this PR/branch specifically - I've had no issues with `gh pr checkout` except with `gh pr checkout 11034`...

**Comment by uncenter** [comment]
Ah, it's working now. I'll review.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3420,6 +3420,11 @@
             "hex": "79BD42",
             "source": "https://comicfury.com/images/gator-icon-black.png"
         },
+        {
+            "title": "Comma",
```
Comment (uncenter) [inline _data/simple-icons.json:3424]:
```suggestion
            "title": "comma",
```

File: icons/comma.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Comma</title><path d="M6.708 24c0-.554-.043-1.02.02-1.47.027-.192.252-.396.437-.51.89-.564 1.856-1.014 2.685-1.659 2.686-2.091 4.277-4.847 4.364-8.425.024-1.014-.349-1.269-1.208-.873-2.47 1.141-5.126.47-6.546-1.651a6.045 6.045 0 0 1 .524-7.419c2.37-2.585 6.263-2.662 8.949-.195 1.611 1.477 2.373 3.387 2.564 5.556.638 7.274-3.02 13.266-9.684 15.9-.658.26-1.333.474-2.105.746"/></svg>
```
Comment (uncenter) [inline icons/comma.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>comma</title><path d="M6.708 24c0-.554-.043-1.02.02-1.47.027-.192.252-.396.437-.51.89-.564 1.856-1.014 2.685-1.659 2.686-2.091 4.277-4.847 4.364-8.425.024-1.014-.349-1.269-1.208-.873-2.47 1.141-5.126.47-6.546-1.651a6.045 6.045 0 0 1 .524-7.419c2.37-2.585 6.263-2.662 8.949-.195 1.611 1.477 2.373 3.387 2.564 5.556.638 7.274-3.02 13.266-9.684 15.9-.658.26-1.333.474-2.105.746"/></svg>
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by 21st-centuryman** [comment]
@uncenter added your changes. 

If there is nothing else for me to change, I hope you have a great weekend.

**Comment by uncenter** [comment]
> @uncenter added your changes.
> 
> If there is nothing else for me to change, I hope you have a great weekend.

I think that is it! Thanks for adding this 🚀

**Review by uncenter — APPROVED** [review]

File: _data/simple-icons.json [hunk 3]
```diff
@@ -3442,6 +3442,11 @@
             "hex": "79BD42",
             "source": "https://comicfury.com/images/gator-icon-black.png"
         },
+        {
+            "title": "comma",
+            "hex": "51FF00",
+            "source": "https://github.com/commaai/connect/blob/d431d977fb79e01a0f565f259a98ec3819b2b3c5/public/icon-vector.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:3448]: Ooh actually I misinterpreted this URL. This isn't an outreach/community repository, it's a specific subproduct. We should use the logo in their footer, https://comma.ai/images/logo-icon-shape.svg. Can you update the source here and verify it still matches with your icon here?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 3 above)
Comment (21st-centuryman) [reply _data/simple-icons.json]: [de23288](https://github.com/simple-icons/simple-icons/pull/11034/commits/de232882dd361e50dd57c98a1f156dd3dfac5a1c)

File: _data/simple-icons.json [hunk 4]
```diff
@@ -3442,6 +3442,11 @@
             "hex": "79BD42",
             "source": "https://comicfury.com/images/gator-icon-black.png"
         },
+        {
+            "title": "comma",
+            "hex": "51FF00",
+            "source": "https://comma.ai/images/logo-icon-shape.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:3448]:
```suggestion
            "source": "https://comma.ai"
```
Given the SVG is in the source code of the homepage, let's tidy this up.

File: icons/comma.svg
Comment (adamrusted) [inline icons/comma.svg:1]: Seeing a bit of a difference (yours in black, source in white) - which might just be a precision thing. Would you mind re-exporting with a higher precision please @21st-centuryman?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/comma.svg
Comment (21st-centuryman) [reply icons/comma.svg:1]: @adamrusted I am using svgo with 5 as percision. what else could i do to fix this icon?

Comment (adamrusted) [reply icons/comma.svg:1]: Thanks for the update @21st-centuryman - I'm _still_ seeing some difference though. Leave it for now and see if another maintainer (@uncenter) can spot the same differences, or if my editing software is on it's way out...

File: icons/comma.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>comma</title><path d="M6.7084 24c0-.55386-.04365-1.02045.02013-1.47025.02686-.19134.25176-.3961.43638-.51023.88954-.56393 1.85628-1.01374 2.6854-1.65823 2.6854-2.09125 4.27649-4.84714 4.36376-8.42543.0235-1.01374-.3491-1.26885-1.20842-.87276-2.47057 1.1413-5.12575.46995-6.54566-1.65151a6.0455 6.0455 0 0 1 .52366-7.4184c2.36986-2.5847 6.26368-2.6619 8.94908-.1947 1.61124 1.47697 2.37322 3.38695 2.56455 5.55541.63778 7.27407-3.02107 13.26586-9.6842 15.9009-.65793.25847-1.33264.4733-2.10469.7452"/></svg>
```
Comment (uncenter) [inline icons/comma.svg:1]: I'm not seeing any differences after saving, optimizing, and reopening with the following:

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><title>comma</title><path d="M6.72682 24c0-.55531-.0434-1.02051.02045-1.46891.0272-.1912.25037-.39595.43237-.51143.88731-.56315 1.85122-1.01275 2.67734-1.65857 2.67584-2.09206 4.26201-4.84612 4.3487-8.42366.02452-1.01583-.34891-1.2696-1.20211-.87389-2.4628 1.1424-5.11119.47263-6.5246-1.65034-1.54137-2.3155-1.32431-5.3984.52253-7.4211 2.359-2.58344 6.24053-2.66074 8.91722-.19346 1.60337 1.47794 2.3652 3.38627 2.5552 5.5569.63691 7.27188-3.01046 13.2657-9.64881 15.89874-.657.26045-1.3307.4744-2.09828.74571"/></svg>
```

File: icons/comma.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" ><title>comma</title><path d="M6.72682 24c0-.55531-.0434-1.02051.02045-1.46891.0272-.1912.25037-.39595.43237-.51143.88731-.56315 1.85122-1.01275 2.67734-1.65857 2.67584-2.09206 4.26201-4.84612 4.3487-8.42366.02452-1.01583-.34891-1.2696-1.20211-.87389-2.4628 1.1424-5.11119.47263-6.5246-1.65034-1.54137-2.3155-1.32431-5.3984.52253-7.4211 2.359-2.58344 6.24053-2.66074 8.91722-.19346 1.60337 1.47794 2.3652 3.38627 2.5552 5.5569.63691 7.27188-3.01046 13.2657-9.64881 15.89874-.657.26045-1.3307.4744-2.09828.74571"/></svg>
```
Comment (adamrusted) [inline icons/comma.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>comma</title><path d="M6.72682 24c0-.55531-.0434-1.02051.02045-1.46891.0272-.1912.25037-.39595.43237-.51143.88731-.56315 1.85122-1.01275 2.67734-1.65857 2.67584-2.09206 4.26201-4.84612 4.3487-8.42366.02452-1.01583-.34891-1.2696-1.20211-.87389-2.4628 1.1424-5.11119.47263-6.5246-1.65034-1.54137-2.3155-1.32431-5.3984.52253-7.4211 2.359-2.58344 6.24053-2.66074 8.91722-.19346 1.60337 1.47794 2.3652 3.38627 2.5552 5.5569.63691 7.27188-3.01046 13.2657-9.64881 15.89874-.657.26045-1.3307.4744-2.09828.74571"/></svg>
```

**Review by adamrusted — APPROVED** [review]
Perfect, thanks @21st-centuryman and @uncenter!

**Pull request merged**